slide/src/test/ui/extra_items_expr_pat.slide

```diff
@@ -0,0 +1,28 @@
+!!!args
+--expr-pat
+!!!args
+
+===in
+_a + $b -#c + 1 2
+3 4 5 6 / 7 ^ 8
+  9
+===in
+
+~~~stdout
+~~~stdout
+
+~~~stderr
+error: Unexpected extra tokens
+  |
+1 |   _a + $b -#c + 1 2
+  |  _________________^
+2 | | 3 4 5 6 / 7 ^ 8
+3 | |   9 
+  | |____^ not connected to a primary expression
+  |
```

Comment:
this error msg is slighly confusing because it connects the two lines.

Reply:
i agree, it might be nicer if we underline the whole thing. I'll look into it..